lib/system/gc_interface.nim

```diff
@@ -36,8 +53,10 @@ when hasAlloc and not defined(js) and not usesDestructors:
     ## sweep phase may take a long time and is not needed if the application
     ## does not create cycles. Thus the mark and sweep phase can be deactivated
     ## and activated separately from the rest of the GC.
-
+  
+  proc GC_getData*(): GCStatsData {.rtl, benign.}
```

Comment:
I had to add GC_getData here, to have it available in the scope of nim.nim. 
Surely, adding methods to this interface means i should also add an mock implementation for every other gc?

Reply:
Modules expose an interface (all exported items), so if we're doing some sort of static interface through conditional compilation then yes that interface should be informed implemented.